ui/src/components/ai-chat/component/chat-input-operate/index.vue

```diff
@@ -452,6 +492,10 @@ function deleteFile(index: number, val: string) {
     uploadImageList.value.splice(index, 1)
   } else if (val === 'document') {
     uploadDocumentList.value.splice(index, 1)
+  } else if (val === 'video') {
+    uploadVideoList.value.splice(index, 1)
+  } else if (val === 'audio') {
+    uploadAudioList.value.splice(index, 1)
   }
 }
 function mouseenter(row: any) {
```

Comment:
The code for the File Upload component has two issues that need attention:

1. You have added new properties `downloadCount` and `imageTotalSize`, which can be removed since these properties should always be returned from methods like 'getFileUploadList()' or an external source.

2. The logic is incorrect when adding files to the list (see point 9 above). The maximum number of allowed images should not just be increased but also decreased each time there's a new file added until it hits zero, rather than using conditional statements (`if (...`). Here's one correct approach:
```javascript
export default {
  name: 'FileUpload',
  data() {
    return {};
```

I recommend fixing this before further implementation steps, such as refactoring and integrating with your existing application architecture and functionality. Consider revising how files actually come through in this context, especially regarding the maximum count of individual types of files. If you're aiming for a clean interface instead of keeping all lists synchronized, creating additional state might help separate concern levels without complicating things too much.